app/AccountancyModule/Components/Cashbook/PrefixControl.php

```diff
@@ -79,15 +75,15 @@ protected function createComponentForm() : BaseForm
 
         $form->addSubmit('submit', 'Uložit');
 
-        $form->onSuccess[] = function ($_, array $values) : void {
```

Comment:
Tohle by pak stálo za to přidat do výjimek - `_` je běžně používanej název pro nepoužitou proměnnou, tak ať nemusíme psát jiný věci.

Reply:
já jsem to všude změnil na $_x. abychom neměli moc vyjímek, ale klidně to předělám, protože to mám taky raději jen $_. Tam to ale hlásí, ze to první písmeno není malé, tedy to nelze vypnout globálně. Takze bys to vypínal jednotlivě?